com.centurylink.mdw.designer.ui/src/com/centurylink/mdw/plugin/project/model/WorkflowProject.java

```diff
@@ -3054,4 +3054,19 @@ else if (url.startsWith("https"))
                 return new DesignerHttpHelper(new URL(url), getUser().getUsername(), getUser().getPassword());
         return new DesignerHttpHelper(new URL(url));
     }
+
+    public void setDevUser() {
+        HttpHelper httpHelper;
+        String url = getServiceUrl() + "/Services/Users?devUser=true";
+        try {
+            httpHelper = new HttpHelper(new URL(url));
+            httpHelper.setConnectTimeout(MdwPlugin.getSettings().getHttpConnectTimeout());
+            httpHelper.setReadTimeout(MdwPlugin.getSettings().getHttpReadTimeout());
+            JSONObject response = new JSONObject(httpHelper.get());
+            user = new User(response.getString("cuid"));
+        }
+        catch (Exception ex) {
+            user = new User(System.getProperty("user.name"));

```

Comment:
I get your intent to fall back to existing behavior if the server is inaccessible.  However, in my experience it is not good practice to silently trap an exception.  I suggest that you consider which type of exception you expect to trigger the fallback behavior (for example IOException).  Even in that case, while I know you want to avoid polluting the log with exceptions due to offline, I suggest you log at least one line to explain that you are falling back to system user.

Feel free to merge this PR yourself after considering whether and how to incorporate my feedback.  Thanks, Don.